src/index.js

```diff
@@ -0,0 +1,87 @@
+const express = require('express')
+const bodyParser = require('body-parser')
+
+const config = require('./config')
+const UserService = require('./services/user')
+const Db = require('./db')
+
+const db = new Db(config.db)
+const userService = new UserService(db)
+const userRouter = express.Router()
+const app = express()
+app.use(bodyParser.json())
+
+/**
+ * @api {get} /user/:id Read the data of a user
+ * @apiName GetUser
+ * @apiGroup User
+ *
+ * @apiParam {Number} id Users unique ID.
+ *
+ * @apiSuccess {String} name          name of the user
+ * @apiSuccess {String} dob           the users date of birth in the format YYYY-MM-DD 
+ * @apiSuccess {String} address       the users address
+ * @apiSuccess {String} description   a description of the user
+ */
+userRouter.get('/:id', (req, res) => {
+  userService.get(req.params.id)
+    .then((user) => {
+      if (user) {
+        res.json(user)
+      } else {
+        res.status(200).send('User not found')
+      }
+    }).catch(e => res.status(400).send(e.message))
```

Comment:
Could you tell me why you prefer 400 status code here?

Reply:
In a proper API I would distinguish between server (`5xx` error codes) (db queries failing, db connection failing, uncaught exceptions etc) and client (`4xx` error codes) failures. In this demo almost all that can go wrong is in data validation so I've just left it as 400.